variables.tf

```diff
@@ -1,130 +1,137 @@
+  type        = number
   description = "CNI interface MTU (only applies to calico and kube-router)"
-  type        = "string"
-  default     = "1500"
+  default     = 1500
+}
+
+variable "network_encapsulation" {
+  type        = string
+  description = "Network encapsulation mode either ipip or vxlan (only applies to calico)"
+  default     = "ipip"
 }
 
 variable "network_ip_autodetection_method" {
+  type        = string
   description = "Method to autodetect the host IPv4 address (only applies to calico)"
-  type        = "string"
   default     = "first-found"
 }
 
 variable "pod_cidr" {
+  type        = string
   description = "CIDR IP range to assign Kubernetes pods"
-  type        = "string"
   default     = "10.2.0.0/16"
 }
 
 variable "service_cidr" {
+  type        = string
   description = <<EOD
 CIDR IP range to assign Kubernetes services.
 The 1st IP will be reserved for kube_apiserver, the 10th IP will be reserved for kube-dns.
 EOD
-
-  type    = "string"
-  default = "10.3.0.0/24"
+  default     = "10.3.0.0/24"
 }
 
-variable "cluster_domain_suffix" {
-  description = "Queries for domains with the suffix will be answered by kube-dns"
-  type        = "string"
-  default     = "cluster.local"
-}
 
 variable "container_images" {
+  type        = map(string)
   description = "Container images to use"
-  type        = "map"
 
   default = {
-    calico           = "quay.io/calico/node:v3.4.0"
-    calico_cni       = "quay.io/calico/cni:v3.4.0"
-    flannel          = "quay.io/coreos/flannel:v0.10.0-amd64"
-    flannel_cni      = "quay.io/coreos/flannel-cni:v0.3.0"
-    kube_router      = "cloudnativelabs/kube-router:v0.2.4"
-    hyperkube        = "k8s.gcr.io/hyperkube:v1.13.2"
-    coredns          = "k8s.gcr.io/coredns:1.3.0"
-    pod_checkpointer = "quay.io/coreos/pod-checkpointer:83e25e5968391b9eb342042c435d1b3eeddb2be1"
+    calico                  = "quay.io/calico/node:v3.13.1"
+    calico_cni              = "quay.io/calico/cni:v3.13.1"
+    coredns                 = "k8s.gcr.io/coredns:1.6.7"
+    flannel                 = "quay.io/coreos/flannel:v0.12.0-amd64"
+    flannel_cni             = "quay.io/coreos/flannel-cni:v0.3.0"
+    kube_apiserver          = "k8s.gcr.io/kube-apiserver:v1.18.1"
+    kube_controller_manager = "k8s.gcr.io/kube-controller-manager:v1.18.1"
+    kube_scheduler          = "k8s.gcr.io/kube-scheduler:v1.18.1"
+    kube_proxy              = "k8s.gcr.io/kube-proxy:v1.18.1"
+    # experimental
+    kube_router = "cloudnativelabs/kube-router:v0.3.2"
   }
 }
 
-variable "enable_reporting" {
-  type        = "string"
-  description = "Enable usage or analytics reporting to upstream component owners (Tigera: Calico)"
-  default     = "false"
-}
 
 variable "trusted_certs_dir" {
+  type        = string
   description = "Path to the directory on cluster nodes where trust TLS certs are kept"
-  type        = "string"
   default     = "/usr/share/ca-certificates"
 }
 
 variable "create_ca" {
```

Comment:
We don't need to support this anymore since we're no longer trying to pass in CA certs for any clusters